### ISSUE TITLE
Fix phottix rig take to jeweler

### ISSUE THREAD

**Issue by CIEspost** [zacck/cieproductions#37]

**Comment by zacck-zz** [comment]
When do you need to have this done, can we look for a jeweller and quotes

**Comment by CIEspost** [comment]
@zacck i dont need this done in the next sprint

**Comment by zacck-zz** [comment]
@CIEspost  then move it to the icebox

**Comment by zacck-zz** [comment]
@CIEspost  what must we do for this?  Find a Jeweller? We can got to the guy we went to with the camera last time they might be able to make it

**Comment by CIEspost** [comment]
@zacck yes maybe we can go to him ja good idea, it is in montagu gardens

**Comment by CIEspost** [comment]
@zacck lets go tomorow please, int he morning, shit I also need to find the tiny screws and other parts that go with the thing

**Comment by zacck-zz** [comment]
@CIEspost  I have put it on my calendar

**Comment by CIEspost** [comment]
cool!
